### PR TITLE
style(sidebar-content): truncate organisation name

### LINKED ISSUE #688: [ENG-131] Org switcher text not truncated
Org switcher: when a user or org's name is long, the text is not truncated

<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-131](https://linear.app/unn/issue/ENG-131/org-switcher-text-not-truncated)</sub>

### PR DESCRIPTION
**Pull request by SySagar**
## What does this PR do?

Fixes #688

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [x] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Tested my code in a local environment
- [x] Commented on my code in hard-to-understand areas
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

https://github.com/user-attachments/assets/460db963-df88-4f10-8aef-b927c8e312f2

### CODE REVIEW

**Comment by BlankParticle** [comment]
image

It would be good if you also truncate the name too

File: apps/web/src/app/[orgShortcode]/_components/sidebar-content.tsx [hunk 1]
```diff
@@ -82,30 +82,37 @@ function OrgMenu() {
...
-
-          <div className={'h-5 w-5 p-0.5'}>
-            <CaretUpDown className={'h-4 w-4'} />
+            <div className={'h-5 w-5 p-0.5 pl-1'}>
+              <CaretUpDown className={'h-4 w-4'} />
+            </div>
           </div>
         </DropdownMenuTrigger>
-        <DropdownMenuContent className="bg-base-1 border-base-5 flex w-fit flex-col gap-0 p-0">
+        <DropdownMenuContent className="bg-base-1 border-base-5 flex w-[350px] flex-col gap-0 p-0">
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortcode]/_components/sidebar-content.tsx:115]: make it `w-fit max-w-[250px]` that would look best from my experiments

**Review by BlankParticle — CHANGES_REQUESTED** [review]
just the width

File: apps/web/src/app/[orgShortcode]/_components/sidebar-content.tsx (hunk 1 above)
Comment (SySagar) [reply apps/web/src/app/[orgShortcode]/_components/sidebar-content.tsx]: done!

**Review by BlankParticle — APPROVED** [review]

**Comment by BlankParticle** [comment]
### Merge activity

* **Aug 15, 6:44 AM EDT**: @BlankParticle started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/un/inbox/693).

**Pull request merged**